### PR TITLE
Enable Stale issues & PRs Workflow

### PR DESCRIPTION
**Pull request by svpernova09**

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  stale:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'Stale issue message'
+        stale-pr-message: 'Stale pull request message'
```
Comment (localheinz) [inline .github/workflows/stale.yml]: Perhaps use more detailed, friendlier messages?

Comment (svpernova09) [reply .github/workflows/stale.yml]: Good call. This was the initial "Hey we got Actions enabled!" PR :D

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  stale:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue has been hanging around a bit too long, if needed please update or comment.'
+        stale-pr-message: 'This PR has been hanging around a bit too long, please update/rebase or comment as needed'
```
Comment (localheinz) [inline .github/workflows/stale.yml]: Are we missing a full stop here?

Comment (svpernova09) [reply .github/workflows/stale.yml]: Not sure what you mean?

**Pull request merged**